libr/core/cmd_debug.c

```diff
@@ -136,7 +136,6 @@ static void dot_trace_traverse(RCore *core, RTree *t) {
 		}
 	}
 	r_cons_printf ("}\n");
-	r_list_free (nodes);
```

Comment:
No, because the api returns a constant list that shouldn't be touched. When you will free the graph it will free also the nodes.